nixos/modules/services/security/oauth2_proxy.nix

```diff
@@ -433,6 +441,15 @@ in
       '';
     };
 
+    reverseProxy = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        In case when running behind a reverse proxy, controls whether headers
+        like <literal>X-Real-Ip</literal> are accepted.
```

Comment:
please rewrite to be more clear about what "accepted" means. Passed through?

Reply:
I just copied the text from their docs: https://oauth2-proxy.github.io/oauth2-proxy/configuration. I'll try to rewrite it to make it more understandable.